src/raftstore/store/store.rs

```diff
@@ -816,7 +816,6 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         assert!(!p.is_applying());
 
         let is_initialized = p.is_initialized();
-        let end_key = enc_end_key(p.region());
         if let Err(e) = p.destroy() {
```

Comment:
No, we need to set a tombstone key. And it's better to let peer itself to decide which part should be done and skipped when it's not initialized.